api/k0smotron.io/v1beta1/k0smotroncluster_types.go

```diff
@@ -84,6 +84,13 @@ type ClusterSpec struct {
 	// https://kubernetes.io/docs/concepts/storage/volumes
 	//+kubebuilder:validation:Optional
 	Manifests []v1.Volume `json:"manifests,omitempty"`
+	// Files allows to specify list of volumes with any files to be
+	// mounted in the controlplane pod. K0smotron allows any kind of volume, but the
+	// recommendation is to use secrets and configmaps.
+	// For more information check:
+	// https://kubernetes.io/docs/concepts/storage/volumes
+	//+kubebuilder:validation:Optional
+	Files []Files `json:"files,omitempty"`
```

Comment:
Could this be called something like `Volumes` or `Mounts`. IMO files sounds bit wrong as we're really not doing files per se directly

Reply:
Yeah, makes sense. Renamed it to `Mounts`, thanks.